src/I18n/Number.php

```diff
@@ -276,6 +276,10 @@ public static function formatter($options = [])
             $type = NumberFormatter::CURRENCY;
         }
 
+        if (!empty($options['type']) && $options['type'] === 'ordinal') {
```

Comment:
There's is no need for a `is_int` check since the `$style` argument of [NumberFormatter::__construct()](http://php.net/manual/en/numberformatter.create.php) for which this `type` option is used is integer.